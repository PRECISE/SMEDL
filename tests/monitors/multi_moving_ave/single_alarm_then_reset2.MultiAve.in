measurement,0,11.0
measurement,0,12.0
measurement,0,1.0
measurement,0,1.0
