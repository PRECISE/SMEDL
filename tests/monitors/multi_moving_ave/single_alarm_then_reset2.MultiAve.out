_Frontend_raiseAlarm
_Frontend_resetAlarm
